blocked-edges/4.15.48-NonZonalAzureMachineSetScaling.yaml

```diff
@@ -0,0 +1,16 @@
+to: 4.15.48
+from: ^4[.](14[.].*|15[.]([1-3]?[0-9]|4[0-7]))[+].*$
+
+url: https://issues.redhat.com/browse/OCPCLOUD-2975
+name: NonZonalAzureMachineSetScaling
+message: |-
+  Azure Clusters in non-zonal regions will fail at scaling MachineSets created in earlier versions.
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      (
```

Comment:
nit: you don't need these entire-expression-wrapping parens.  They don't hurt either, and you do need them if you want to join "am I an Azure cluster?" against other conditions like "am I an OVN cluster?", but pointing this out in case you want to drop two lines this time around.  I'm fine if you want to leave it as you have it too.